recipes/jupyterlab-gather/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab-gather" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_gather-{{ version }}.tar.gz
+  sha256: 1246a16edb896676bff19e2ec8e05779cbb7fdfb7af30e0fba894d2bc4a0e2c2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
```

Comment:
It looks like you may need `hatch-jupyter-builder >=0.5` in the `host` section.

Reply:
Thanks!